src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/Extensions/LogLevelExtension.cs

```diff
@@ -34,5 +34,41 @@ internal static string ToStringOptimized(this LogLevel logLevel)
                     return logLevel.ToString(CultureInfo.InvariantCulture);
             }
         }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static bool TryParseOptimized(this string logLevel, out LogLevel level)
+        {
+            switch (logLevel)
```

Comment:
Switch is case sensitive by default. Do we have to worry about case?

Reply:
Actually, I researched this. Enum.TryParse which we are using now is also case sensitive by default. I looked into some true case insensitive options, but they all 10x'd the time cost of the method, even some really cool dictionary methods. I could just add the all lower equivalents to the case statement, but I figure since this behavior is the same as the existing method, no need.